src/components/split-button/split-button.tsx

```diff
@@ -0,0 +1,90 @@
+import { Component, Event, EventEmitter, Host, h, Prop } from '@stencil/core';
+import { ListSeparator, MenuItem } from '@limetech/lime-elements';
+
+/**
+ * A split button is a button with two components:
+ * a button and a side-menu attached to it.
+ *
+ * Clicking on the button runs a default action,
+ * and clicking on the arrow opens up a list of other possible actions.
+ *
+ * :::warning
+ * - Never use a split button for navigation purposes, such as going to next page.
+ * The button should only be used for performing commands!
+ * - Never use this component instead of a Select or Menu component!
+ * :::
```

Comment:
No _must_, and can be polished later too, but it might be nice to have some example that's a little easier to understand for these "warnings". Like the DOs and DON'Ts that the Material Design docs use, or something.

Also, it might be worth adding something about how it's good to collect "similar" actions under a split button (like _Save_, and _Save As_), but not to collect "dissimilar" actions (like _Save_ and _Delete_).